docker/gaffer/docker-compose.yaml

```diff
@@ -157,3 +157,15 @@ services:
     - 8080:8080
     volumes:
     - ./conf/store.properties:/gaffer/store/store.properties:ro
+
+  gaffer-ui:
```

Comment:
So if I understand, this means that the gaffer docker-compose will also build the ui and point it to the rest? Do the docs need updating?

Reply:
Correct - both the REST API and UI used to both be deployed but it looks like when they were split out into separate components the UI was omitted by accident. This puts it back and good call on the docs - I've updated the link to the UI as it's now available on a different port.